src/main/cpp/t03_circle.cpp

```diff
@@ -38,8 +38,13 @@
 using namespace std;
 
 //function IsPointInCircle
-
+bool isPointInCircle(double x, double y, double xc, double yc, double r) {
+	bool isPointInCircle = pow((x - xc), 2) + pow((y - yc), 2) <= pow(r,2);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'isPointInCircle' is assigned a value that is never used.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4594246)